tutorecommerce/templates/ecommerce/apps/tutor.py

```diff
@@ -61,6 +78,22 @@
 COMPRESS_OFFLINE = True
 
 PAYMENT_PROCESSOR_CONFIG = {
-    "openedx": json.loads("""{{ ECOMMERCE_PAYMENT_PROCESSORS|tojson(indent=4) }}""")
+    "edx": json.loads("""{{ ECOMMERCE_PAYMENT_PROCESSORS|tojson(indent=4) }}""")
 }
-PAYMENT_PROCESSORS = list(PAYMENT_PROCESSORS) + {{ ECOMMERCE_EXTRA_PAYMENT_PROCESSOR_CLASSES }}
\ No newline at end of file
+PAYMENT_PROCESSORS = list(PAYMENT_PROCESSORS) + {{ ECOMMERCE_EXTRA_PAYMENT_PROCESSOR_CLASSES }}
+
+SCAR_DEFAULT_CURRENCY = 'CNY'
+CURRENCY_SYMBOL = '¥'
+ENABLE_ALIPAY_WECHATPAY = True
```

Comment:
Unfortunately, we cannot add alipay-specific settings for each user. But I do not think that you require such settings. Let's talk about it by chat?